sys/arm64/arm64/swtch.S

```diff
@@ -253,20 +253,33 @@ ENTRY(fork_trampoline)
 	msr	daifset, #2
 
 	/* Restore sp and lr */
+#if __has_feature(capabilities)
+	ldp	c0, c1, [sp, #TF_SP]
```

Comment:
#ifdef could be avoided here if we had REG()/CREG()/PTR()/something else macros.